source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst

```diff
@@ -104,7 +104,9 @@ Guidance
 Publishing and Subscribing
 --------------------------
 
-The driver SHOULD publish events in a manner that is standard to the driver's language publish/subscribe patterns and is not strictly mandated in this specification.
+The driver SHOULD publish events in a manner that is standard to the driver's
```

Comment:
There appear to be a few white space changes in this file. Can you please revert them so that we can focus on what's actually changing in the spec?

Reply:
Done.